packages/language-service/lib/language-plugin.js

```diff
@@ -34,13 +35,13 @@ export function createMdxLanguagePlugin(
   processor.freeze()
 
   return {
-    getLanguageId(scriptId) {
-      if (scriptId.endsWith('.mdx')) {
+    getLanguageId(fileNameOrUri) {
+      if (String(fileNameOrUri).endsWith('.mdx')) {
```

Comment:
This is fine, but it feels weird. Wouldn’t it be better if the TypeScript integration converts the file name to a file URI?

Reply:
I have also considered that, but the cost is too high for non-LSP runtime, because it requires converting the URI every time before calling `language.scripts.get`, and unnecessarily adding a dependency on `vscode-uri`.